tests/gtest/avifgainmaptest.cc

```diff
@@ -1358,7 +1358,9 @@ TEST(FindMinMaxWithoutOutliers, AllSame) {
     std::vector<float> values(kNumValues, v);
 
     float min, max;
-    avifFindMinMaxWithoutOutliers(values.data(), kNumValues, &min, &max);
+    ASSERT_EQ(
+        avifFindMinMaxWithoutOutliers(values.data(), kNumValues, &min, &max),
```

Comment:
Maybe make that function nodiscard?

Reply:
It returns un avifResult so it's already  nodiscard.
Not sure why it didn't show up in presubmit.